dfsclient/src/main/java/org/hpccsystems/dfs/client/FileUtility.java

```diff
@@ -660,6 +679,44 @@ public void run()
         }
     }
 
+    private static Runnable[] createReadTestTasks(DataPartition[] fileParts, FieldDef recordDef, TaskContext context) throws Exception
+    {
+        Runnable[] tasks = new Runnable[fileParts.length];
+        for (int i = 0; i < tasks.length; i++)
+        {
+            final int taskIndex = i;

```

Comment:
Minor, but could "taskIndex" be declared in the for construct instead of "i"?

Reply:
Good question, that taskIndex has to be final to allow the runner to be able to use it. Which is why it is declared in the loop vs being used as the loop counter.